### ISSUE TITLE
Mipmap levels limited to 0

### ISSUE THREAD

**Issue by EthanRStokes** [Jack-Bagel/End-Remastered#3]
The black eye texture limits the mip level to 0

From the log:
```
[14:43:11]  [Worker-ResourceReload-8/WARN]:Texture endrem:block/eyes/black_eye with size 32x11 limits mip level from 4 to 0
```

Version: 6.0.0
MC Version: 1.21

**Comment by Jack-Bagel** [comment]
How big of an issue is this?

**Comment by EthanRStokes** [comment]
> How big of an issue is this?

I'd say fairly big for most. This effectively forces the Mipmap Levels option in video settings to 0, making the world look rather ugly.

Distant textures look very bad.
![image](https://github.com/user-attachments/assets/8e66513b-40f6-4da4-b3d4-68d1e8c20942)

**Comment by Jack-Bagel** [comment]
I thought the issue only affected the eyes rendering, not the whole game lol. It will be fixed in the next update.

**Comment by Jack-Bagel** [comment]
Fixed with commit 22171e4
